### ISSUE TITLE
Allow use of range values for alternatives

### ISSUE THREAD

**Issue by mamins1376** [illicitonion/num_enum#101]
Currently (`0.5.9`), trying to use any form of range expressions for `alternatives` attribute results in the following error:
```rust
error: Invalid attribute: expected `,`
  --> src/main.rs:
   |
   |         #[num_enum(alternatives = [20..100])]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Without this feature, covering large ranges (like above) is practically impossible. Allowing for use of range expressions (bounded or unbounded, exclusive or inclusive) would make this a lot easier and will be quite useful, particularly in protocol definitions for network applications or any kind of declarative pattern matching, where the range data and `enum` definition will be located at the same place.

Thanks for this great crate!

**Comment by illicitonion** [comment]
I think this feature request makes sense - I've got a working implementation locally, I just need to clean it up a little, will get it put up soon!

**Comment by illicitonion** [comment]
Released in [0.5.11](https://crates.io/crates/num_enum/0.5.11) - hope it's useful!
